content/projects/linux/advanced/_index.md

```diff
@@ -3,7 +3,7 @@ _db_id: 187
 content_type: project
 flavours:
 - bash
-ready: false
+ready: true
 submission_type: repo
 title: Advanced Linux challenges
 todo: 2.4 doesn't make sense
```

Comment:
Apparently 2.4doesn't make sense. So that means this probably is not ready. 

If this isn't being used anywhere you can leave it as ready:false.

If it is being used then we should only mark it as ready when it is actually ready.